### ISSUE TITLE
W tensorflow/core/framework/op_kernel.cc:1273] OP_REQUIRES failed at save_restore_v2_ops.cc:184 : Not found: Key squeezenet/Bottleneck/BatchNorm/beta not found in checkpoint

### ISSUE THREAD

**Issue by 8YY2** [boyliwensheng/understand_facenet#8]
你好.....在把model改成squeezenet的时候出现了这个error....想问一下怎么解决 【如果不挂官方预训练模型好像就不会报错
